dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -37,39 +37,42 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
   navItems = [
     {
       key: 'overview',
-      icon: 'mdi mdi-view-dashboard-outline'
+      icon: 'mdi mdi-view-dashboard-outline',
     },
     {
       key: 'data',
-      icon: 'mdi mdi-file-document-box-outline'
+      icon: 'mdi mdi-file-document-box-outline',
     },
     {
       key: 'verifications',
-      icon: 'mdi mdi-checkbox-marked-circle-outline'
+      icon: 'mdi mdi-checkbox-marked-circle-outline',
     },
     {
       key: 'sharings',
-      icon: 'mdi mdi-share-variant'
+      icon: 'mdi mdi-share-variant',
     },
     {
       key: 'did',
-      icon: 'mdi mdi-identifier'
-    }
-  ].map(entry => {
-    return {
-      label: `_twin-detail.${entry.key}.${entry.key}-title`,
-      icon: entry.icon,
-      to: { name: entry.key }
-    };
-  });
+      icon: 'mdi mdi-identifier',
+    },
+  ].map((entry) => ({
```

Comment:
seems bit over-engineered to built the object-array from the array here, could be as easily written directly or loaded from a json. Would also offer the flexibility to use different translation keys, independent from `key`